lib/rspecq/queue.rb

```diff
@@ -241,6 +242,22 @@ def flaky_jobs
       requeued - @redis.hkeys(key_failures)
     end
 
+    def fail_fast_threshold
```

Comment:
This should be a simple `attr_reader` instead, and the default should be set in `#initialize`, like we do with the other settings in `Worker#initialize`.

Reply:
dd69a2a